util/tangy.utils.js

```diff
@@ -0,0 +1,76 @@
+export function _generateObjectId(element) {
```

Comment:
The need to generate an object id is a) common to all form elements and b) would be done pretty much the same way. This function can live.